crates/stc_ts_type_checker/tests/tsc-stats.rust-debug

```diff
@@ -1,6 +1,6 @@
 Stats {
-    required_error: 3535,
-    matched_error: 6500,
-    extra_error: 771,
-    panic: 74,
+    required_error: 874,
+    matched_error: 1749,
+    extra_error: 284,
+    panic: 20,
```

Comment:
Can you run check.sh agian?

Reply:
I got `thread 'conformance::types::uniqueSymbol::uniqueSymbolsDeclarations.ts' has overflowed its stack`
Should i pass some option to avoid the overflow?

Reply:
The way is simple

1. Check that the code you wrote affects stackoverflow.
2. If it does, insert some stackguards.
3. if it doesn't, it's broken and you can report it in an issue

Thanks you